src/feature.js

```diff
@@ -102,18 +79,20 @@ class FxABrowserFeature {
   }
 
   unverifiedUser() {
+    this._state = "unverified";
     this.standardAvatar(UNVERIFIED_AVATAR);
-    browser.browserAction.setPopup({ popup: "popup/unverified/unverified.html" });
+    browser.browserAction.setPopup({ popup: UNVERIFIED_PAGE });
   }
 
   verifiedUser(user) {
-    if (user.avatar) {
+    this._state = "verified";
+    if (user.avatar && !user.avatarDefault) {
```

Comment:
What are the conditions under which `user.avatar` and `user.avatarDefault` are both set? I didn't see that in local testing yet, interested just to understand and reproduce.